app/models/manageiq/providers/ansible_tower/inventory/persister/automation_manager.rb

```diff
@@ -1,19 +1,18 @@
 class ManageIQ::Providers::AnsibleTower::Inventory::Persister::AutomationManager < ManagerRefresh::Inventory::Persister
+  include ManagerRefresh::Inventory::AutomationManager
+
+  has_inventory automation_manager.credentials
```

Comment:
ok, just joined you all. `automation_manager` is hardcoded to this class. and `configuration` is a hash that is ~hardcoded to this class. It contains no instance data.

Just double checking - are we sure `@automation_manager` is different for each parent class?
Think this looks good.